api/src/main/resources/workbench-api.yaml

```diff
@@ -7787,3 +7814,29 @@ definitions:
           timestamp (epoch millis) at workspace creation
         type: integer
         format: int64
+  EditUserInformationRequest:
+    description: >
+      A group of changes an admin can make to a user in the Edit Information card of the individual-user admin page.
+      Updates to the contact email or the institutional affiliation will be rejected if they do not validate.
+    type: object
+    required:
+      - username
+    properties:
+      username:
+        description: The full system-assigned ID of a user, including email domain
+        type: string
+      freeCreditsLimit:
+        description: When set, overrides the default free credits dollar limit for this user.
+        type: number
+        format: double
+      contactEmail:
+        description: When set, replaces the user's contact email.  Must validate against the user's Institutional Affiliation, if one exists.
+        type: string
+      verifiedInstitutionalAffiliation:
```

Comment:
Can we just name this property `affiliation` with the intention to rename `VerifiedInstitutionalAffiliation` soon?

Reply:
I like that.

We should remove all of the old-style affiliations before doing so, but I think we can get to this soon.  New ticket: https://precisionmedicineinitiative.atlassian.net/browse/RW-5326